ext/skywalking/dashboard/dolphinscheduler.yml

```diff
@@ -0,0 +1,284 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# UI templates initialized file includes the default template when the SkyWalking OAP starts up at the first time.
+#
+# Also, SkyWalking would detect the existing templates in the database, once they are missing, all templates in this file
+# could be added automatically.
+
+templates:
+  - name: Dolphinscheduler
+    type: "DASHBOARD"
+    configuration: |-
+      [
+        {
+          "name":"Dolphinscheduler",
+          "type":"service",
+          "serviceGroup":"dolphinscheduler",
+          "children":[
+            {
+              "name":"Master Scheduler",
+              "children":[
+                {
+                  "width":3,
+                  "title":"QuartzScheduler Load",
+                  "height":350,
+                  "entityType":"Endpoint",
+                  "independentSelector":true,
+                  "metricType":"REGULAR_VALUE",
+                  "metricName":"endpoint_cpm",
+                  "queryMetricType":"readMetricsValues",
+                  "chartType":"ChartLine",
+                  "currentService":"dolphinscheduler::master-server",
```

Comment:
Is the service name for skywalking must be `dolphinscheduler::master-server` ?

Reply:
In this pull request we only provide the dashboard for `master-server`, users can still add more dashboard for worker or other components.

The name `DolphinScheduler::master-server` is specified via https://github.com/kezhenxu94/dolphinscheduler/blob/7392b014aa773f9253f325c63bb3bc050c4148bb/script/dolphinscheduler-daemon.sh#L99

so yes its name must be `dolphinscheduler::master-server`